docs/IssueManagementPolicies.md

```diff
@@ -0,0 +1,27 @@
+# Issue Management Policies
+
+We have a lot of issue traffic to manage, so we have a few policies in place to help us do that. This is a brief summary of some of the policies we have in place and the justification for them.
+
+## Commenting on closed issues
+
+In general, we recommend you open a new issue if you have a bug, feature request, or question to discuss. If you find a closed issue that is related, open a *new issue* and link to the closed issue rather than posting on the closed issue. Closed issues don't appear in our triage process, so only the people who have been active on the original thread will be notified of your comment. A new issue will get more attention from the team.
+
+*In general* we don't mind getting duplicate issues. It's easier for us to close duplicate issues than to discuss multiple root causes on a single issue! We may close your issue as a duplicate if we determine it has the same root cause as another. Don't worry! It's not a waste of our time!
```

Comment:
>  Don't worry! It's not a waste of our time!

I put this in because I've have people apologize for "wasting our time" when I mark an issue of duplicate and I'd like to try and make clear that we don't feel it's a waste of our time at all 😄.